styles/home.css

```diff
@@ -0,0 +1,81 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+#home-page {
+  background-color: #f4f4f4;
+  padding-bottom: 100px;
+}
+
+.hello-world {
+  /* margin-bottom: 10px; */
+  padding-top: 60px;
+  font-size: 24px;
+  color: #ec5242;
+}
+
+.h1-line > span {
+  display: block;
+  text-transform: uppercase;
+  font-size: 36px;
+  color: #ec5242;
+  font-weight: 900;
+}
+
+.summary-block {
+  border: 1px solid #fff;
+  padding: 20px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.summary-block > p {
+  font-weight: 400;
+  color: #333;
+}
+
+.date {
+  font-size: 32px;
+  color: #575757;
+  font-weight: 700;
+}
+
+.venue {
+  color: #333;
+}
+
+/* Mobile css - 768px and below */
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] Same here, the CSS styling of this project should be built as a mobile-first.
Please fix it, media queries are supposed to be used in this project to change the design from mobile to desktop, not the other way around.